book/src/internals/name-resolving.md

```diff
@@ -50,7 +50,7 @@ three things can happen:
 
 ## Translating to SQL
 
-When translating into a SQL statement which references only one table, there is
+When translating into an SQL statement which references only one table, there is
```

Comment:
Sorry to leave a comment criticizing the tiniest piece of this PR — but I think `a SQL` would be idiomatic. Or do others have thoughts?

Reply:
OK amazing, there's actually a whole debate out there! https://www.quora.com/Is-it-a-SQL-query-or-an-SQL-query

Reply:
Yeah, it depends on pronunciation. I'm saying S-Q-L, not Sequel, that's why this felt like a typo.

Reply:
I would also say Sequel but I've heard Aljaz prefer to say S-Q-L during our presentation dry runs. 😄

Reply:
Saying "Sequel" is arguably more consistent with saying "Prequel"...!